libafl/src/schedulers/accounting.rs

```diff
@@ -266,15 +266,9 @@ where
             return Ok(());
         }
 
-        state
-            .corpus()
-            .get(idx)?
-            .borrow_mut()
-            .metadata_map_mut()
-            .insert(AccountingIndexesMetadata::with_tcref(
-                indexes,
-                new_favoreds.len() as isize,
-            ));
+        state.current_testcase_mut()?.metadata_map_mut().insert(
```

Comment:
this part is wrong?
it's not equivalent as before.

Reply:
Aah thanks!